azkaban-exec-server/build.gradle

```diff
@@ -4,6 +4,7 @@ dependencies {
     compile(project(':azkaban-common'))
 
     compile deps.kafkaLog4jAppender
+    compile deps.gobblinKafka
```

Comment:
I assume we need to use the Linkedin internal Kafka when used inside linkedin.
How would that work?

Reply:
kafka library used by gobblin metrics works fine within LinkedIn. We are using in metric store as well and did not observe any issues by using this.

Reply:
Could you double check with the Kafka and Gobblin team? The open source version may work. But I thought the internal version has to be used. It's good to be prepared even if we can use the open source version for now.

Reply:
I have checked this with Aditya in the gobblin team and he confirmed that this is the right way to use it.